cves/kernel/CVE-2021-46283.yml

```diff
@@ -472,7 +505,12 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    The mistakes that contributed to this vulnerability were coding mistakes that could 
+    also invovle design, maintainability, and miscommunicaitons. They all could be classified as
+    a lapse as it was a simple memory allocation mistake that became an issue after modifying and 
+    introducing new features/code many times. This is why I say maintainability and communcation, among
```

Comment:
maybe avoid first-person perspective and build a better argument

Reply:
agreed